packages/gatsby-plugin-mdx/gatsby/source-nodes.js

```diff
@@ -184,7 +184,8 @@ module.exports = (
             return html
           } catch (e) {
             reporter.error(
-              `Error querying the \`html\` field. This field is intended for use with RSS feed generation.
+              `gatsby-plugin-mdx: Error querying the \`html\` field.
```

Comment:
@pieh Shouldn't `reporter.error` already mention the plugin it was called in?

Reply:
Never mind. It will only after https://github.com/gatsbyjs/gatsby/issues/19024 is resolved 

This is okay for now